outputs/kubeless.go

```diff
@@ -72,6 +80,11 @@ func (c *Client) KubelessCall(falcopayload types.FalcoPayload) {
 		}
 		log.Printf("[INFO]  : Kubeless - Function Response : %v\n", string(rawbody))
 	} else {
+		c.AddHeader(KubelessEventIDKey, uuid.New().String())
+		c.AddHeader(KubelessEventTypeKey, KubelessEventTypeValue)
+		c.AddHeader(KubelessEventNamespaceKey, c.Config.Kubeless.Namespace)
+		c.ContentType = KubelessContentType
```

Comment:
Compare to the following hunk in the current main branch:
https://github.com/falcosecurity/falcosidekick/blob/970a90504eb3d761c8cc9617a9153a9397ba5dbf/outputs/client.go#L165-L168